src/pages-and-resources/discussions/DiscussionsSettings.test.jsx

```diff
@@ -191,6 +191,29 @@ describe('DiscussionsSettings', () => {
     });
   });
 
+  describe('Check Stepper error message', () => {
+    beforeEach(async () => {
+      axiosMock.onGet(getAppsUrl(courseId)).reply(200, legacyApiResponse);
+      renderComponent();
+      history.push(`/course/${courseId}/pages-and-resources/discussion`);
+      await waitForElementToBeRemoved(screen.getByRole('status'));
+      userEvent.click(queryByLabelText(container, 'Select edX'));
+      userEvent.click(queryByText(container, appListMessages.nextButton.defaultMessage));
+    });
+    test('incomplete setting error message appears/disappears on stepper',
+      async () => {
+        await store.dispatch(updateValidationStatus({ hasError: true }));
+        const stepper = container.getElementsByClassName('pgn__stepper-header-step-active');
```

Comment:
again, this shouldn't depend upon paragon classes.

Reply:
then I will need to add a data-testid because there is no selector available for the stepper component.

Reply:
Yes, we can add testids if that's absolutely necessary.